dll/win32/comdlg32/lang/cdlg_Bg.rc

```diff
@@ -160,10 +160,10 @@ FONT 8, "MS Shell Dlg"
  COMBOBOX cmb1, 6, 114, 90, 60, CBS_DROPDOWNLIST | CBS_AUTOHSCROLL | WS_BORDER | WS_VSCROLL | WS_TABSTOP
  LTEXT "&Устройства:", stc4, 110, 104, 92, 9
  COMBOBOX cmb2, 110, 114, 92, 68, CBS_DROPDOWNLIST | CBS_OWNERDRAWFIXED | CBS_AUTOHSCROLL | CBS_SORT | CBS_HASSTRINGS | WS_BORDER | WS_VSCROLL | WS_TABSTOP
- DEFPUSHBUTTON "Отвори", IDOK, 208, 6, 60, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
+ DEFPUSHBUTTON "Отвори", IDOK, 208, 6, 60, 14, WS_GROUP | WS_TABSTOP
  PUSHBUTTON "Отмени", IDCANCEL, 208, 24, 60, 14, WS_GROUP | WS_TABSTOP
  PUSHBUTTON "&Помощ", pshHelp, 208, 46, 60, 14, WS_GROUP | WS_TABSTOP
- CHECKBOX "Само за &четене", chx1, 208, 68, 65, 12, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
+ AUTOCHECKBOX "Само за &четене", chx1, 208, 68, 65, 12, WS_GROUP
```

Comment:
Are you sure, that the behavior is not changed if you omit the `WS_TABSTOP` here? `WS_TABSTOP` is set by default, when you use an AUTOCHECKBOX and if you do not set any more style explicitly. But since you do set a `WS_GROUP` here explicitly, that may mean that you have to assign here both `WS_GROUP | WS_TABSTOP` to not lose the `WS_TABSTOP`

Reply:
I have tested with a code example and verify with ResourceHacker, that flag operations are always an OR operation.

Reply:
That's okay for me to not block your changes. I will not collapse the discussion yet, so that @HBelusca can see it as well. Because I do have some remaining doubts.

Reply:
There's a good chance that the WS_TABSTOP is removed/Not present when any other WS is present (e.g. WS_GROUP)

> If you **do not specify a style**, the default style is BS_AUTOCHECKBOX | WS_TABSTOP.

At least I've noticed that with other controls when I did my own WS cleanup PRs

Reply:
not really. Needs more accurate test.

Reply:
Send a before and an after screenshot of resource hacker of that exact AUTOCHECKBOX for bg-BG as a proof.
Make sure that not just the pic of the dlg, but the text above is visible which allows to see the resulting flags.
@maj113 and I do both fear the exact same thing to happen. His memory 100% matches to my personal memory.